configs/testdata/warning-files/redundant_interp.tf

```diff
@@ -0,0 +1,36 @@
+# It's redundant to write an expression that is just a single template
+# interpolation with another expression inside, like "${foo}", but it
+# was required before Terraform v0.12 and so there are lots of existing
+# examples out there using that style.
+#
+# We are generating warnings for that situation in order to guide those
+# who are following old examples toward the new idiom.
+
+variable "triggers" {
+  type = "map" # WARNING: Quoted type constraints are deprecated
```

Comment:
I greatly appreciate your choice to indicate the expected message in the test file, future developers (and me right now) thank you!